predict_bias.py

```diff
@@ -188,8 +192,8 @@ def column(var):
 
 if inc_sfg == 'output':
     ## sample file:
-    file_i = xr.open_dataset('/scratch1/NCEPDEV/stmp2/Tse-chun.Chen/anal_inc/IFS_replay/2020010100/control/INPUT/fv3_increment6.nc')
-    logging.info("saving to fv3_increment6.nc")
+    file_i = xr.open_dataset('/scratch2/BMC/gsienkf/Laura.Slivinski/model_error_corr_work/%s/fv3_increment6.nc'%indate) #later, make this more general
```

Comment:
can we pass the output dir on the command line? we can make `./` to be the default.

Reply:
"file_i" is not the output dir, I did add the output dir as a command line argument. "file_i" is the "sample" increment that the code takes to generate all the variables and metadata before copying it to a new increment file and writing over the values. I could make this a separate command line argument though (just starts to be a very long list of long directory arguments).

Reply:
right. can we assume that fv3_increment6.nc is located in the output directory?

Reply:
Oh I see. Sure.